### PR TITLE
Improve to avoid typos in DP names, possibility to keep latest value, change in startValidity

### PR DESCRIPTION
**Pull request by chiarazampolli**
@shahor02 
Now there are some enum's used. It is not the best probably, but the other option is to change the GRPLHCInfo struct to have a map of vectors where the key is the alias. The disadvantage is to lose direct access to the different information, while becoming closer to teh Collimators. What do yo think?

### CODE REVIEW

**Comment by shahor02** [comment]
Thanks! Enums are fine, the access via map is much slower.

**Comment by chiarazampolli** [comment]
Ok, then let's see what the CI says. 
Note that the default is now to keep the latest measurement in the vectors, but with an option, it can be changed (`--clear-vectors`)

**Comment by chiarazampolli** [comment]
Moving to WIP, I want to change the way the startValidity is determined, using the earlier timestamp in the vectors for EnvVariables, Collimators etc.

**Comment by chiarazampolli** [comment]
CI failure not related to this PR. Let's see the rest.

**Comment by chiarazampolli** [comment]
Hi @shahor02 , do yo think you could review this one?

File: Detectors/GRP/calibration/src/GRPDCSDPsProcessor.cxx [hunk 1]
```diff
@@ -230,55 +277,62 @@ bool GRPDCSDPsProcessor::processLHCIFDPs(const DPCOM& dpcom)
...
+      if (aliasStr == static_cast<std::string>(GRPLHCInfo::bptxPhaseRMSAliases[ibeam])) {
+        updateVector(dpid, mLHCInfo.mBPTXPhaseRMS[ibeam], aliasStr, dpcomdata.get_epoch_time(), val);
         return true;
-      } else if (aliasStr == fmt::format("{}{}", "BPTX_Phase_Shift_B", ibeam)) {
-        updateVector(dpid, mLHCInfo.mBPTXPhaseShift[ibeam - 1], aliasStr, dpcomdata.get_epoch_time(), val);
+      }
+    }
+    for (int ibeam = 0; ibeam < GRPLHCInfo::BPTXPhaseShiftAliases::NBPTXPhaseShiftAliases; ++ibeam) {
+      if (aliasStr == static_cast<std::string>(GRPLHCInfo::bptxPhaseShiftAliases[ibeam])) {
+        LOG(info) << "aliasStr = " << aliasStr << " alias to check = " << static_cast<std::string>(GRPLHCInfo::bptxPhaseShiftAliases[ibeam]);
```
Comment (shahor02) [inline Detectors/GRP/calibration/src/GRPDCSDPsProcessor.cxx]: this one is logged even in non-verbose mode, could you just suppress it?  Will merge w/o another set of CIs

Comment (chiarazampolli) [reply Detectors/GRP/calibration/src/GRPDCSDPsProcessor.cxx]: Ah sorry, leftover, I will fix it.

Comment (chiarazampolli) [reply Detectors/GRP/calibration/src/GRPDCSDPsProcessor.cxx]: Done

**Review by shahor02 — APPROVED** [review]
Tests were passed, afterwards just 1 LOG.. was suppressed, merging

**Pull request merged**